Implement My bike landing page

anastasiia-tilikina opened the pull request:
- [DEMO LINK](https://anastasiia-tilikina.github.io/layout_miami/)

liliia-livandovskaya reviewed (changes requested):
1. Add the logo to the favicon
<img width="90" alt="Знімок екрана 2022-08-26 о 11 44 20" src="https://user-images.githubusercontent.com/104769168/186865093-e6545810-1e2e-463c-b3e6-1792bdb39da8.png">
2. Your form fields should be reset after  submitting
<img width="1021" alt="Знімок екрана 2022-08-26 о 11 45 16" src="https://user-images.githubusercontent.com/104769168/186865375-4eccc9b4-618b-4393-9c84-e39d95f9daef.png">

anastasiia-tilikina commented:
I just added another fivicon, because it seemed me nicer)
But ok, I’ll change it)

and there was a requirement to prevent reloading when submitted, wasn't it??

anastasiia-tilikina commented:
This is the final version of my project prepared for the portfolio.
I have refactored the code using the mobile-first approach and added a project description.

Could you please check it and also have a look at my [README](https://github.com/anastasiia-tilikina/layout_miami/blob/my-bike/README.md) file and leave your comments if needed 🙏
